visualstudio-dev/azuredeploy.json

```diff
@@ -0,0 +1,238 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageName": {
+      "type": "string",
+      "metadata": {
+        "description": "Globally unique storage account name"
+      }
+    },
+    "storageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Standard_LRS"
+      ],
+      "metadata": {
+        "description": "Which type of storage you want to use"
+      }
+    },
+    "DeployLocation": {
+      "type": "string",
+      "defaultValue": "West US",
```

Comment:
As a best practice, please let the users pick a location rather than defaulting to one